kasa/smart/smartdevice.py

```diff
@@ -238,6 +250,13 @@ async def _initialize_features(self):
             for feat in module._module_features.values():
                 self._add_feature(feat)
 
+    @property
+    def is_cloud_connected(self):
+        """Returns if the device is connected to the cloud."""
+        if "CloudModule" not in self.modules:
+            return False
```

Comment:
As this will be part of the public API, we want to have a test coverage for its behavior.

Reply:
Would you mind adding one also for the case of missing cloud module? Otherwise this is ready now.